apps/studio/hooks/analytics/useLogsPreview.tsx

```diff
@@ -42,11 +43,17 @@ interface LogsPreviewHook {
   setFilters: (filters: Filters | ((previous: Filters) => Filters)) => void
   setParams: Dispatch<SetStateAction<LogsEndpointParams>>
 }
-function useLogsPreview(
-  projectRef: string,
-  table: LogsTableName,
+function useLogsPreview({
+  projectRef,
```

Comment:
just making the arguments here into an object so its easier / more flexible to call this hook